src/main/java/com/metamx/collections/bitmap/WrappedImmutableConciseBitmap.java

```diff
@@ -52,7 +53,10 @@ public ImmutableConciseSet getBitmap()
   @Override
   public boolean get(int value)
   {
-    return bitmap.get(value) > 0;
+    final ConciseSet conciseSet = new ConciseSet();
+    conciseSet.add(value);
+    final ImmutableConciseSet immutableConciseSet = ImmutableConciseSet.newImmutableFromMutable(conciseSet);
+    return ImmutableConciseSet.intersection(this.bitmap, immutableConciseSet).size() != 0;
```

Comment:
Yes, after  performing benchmarks to compare the two methods while the first one performs an intersection and the other one iterates over the bitmap, results revealed that the one performing an intersection was about 3 times more efficient as it only reads CPU words without accessing to their set bits, where the other one does more stuff by reading CPU words and extracting their set bits wich leads to significant latencies. Link to the benchmarks: https://bitbucket.org/samytto/bytebuffercollections_benchgetmethods/src/89b0fc17e9faf39413099661464b4f2057e7da0f/src/test/java/com/metamx/collections/bitmap/BitmapBenchmark.java?at=master&fileviewer=file-view-default#BitmapBenchmark.java-153

My results:
data generated. Nb data = 100000
Running com.metamx.collections.bitmap.RangeBitmapBenchmarkTest
RangeBitmapBenchmarkTest.timeGetIterator: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 2.30 [+- 0.01], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 7.02, time.warmup: 2.43, time.bench: 4.60
RangeBitmapBenchmarkTest.timeGetIntersect: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 0.59 [+- 0.00], round.block: 0.00 [+- 0.00], round.gc: 0.01 [+- 0.01], GC.calls: 0, GC.time: 0.00, time.total: 1.84, time.warmup: 0.66, time.bench: 1.19
data generated. Nb data = 100000
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 8.993 sec - in com.metamx.collections.bitmap.RangeBitmapBenchmarkTest
Running com.metamx.collections.bitmap.UniformBitmapBenchmarkTest
UniformBitmapBenchmarkTest.timeGetIterator: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 2.30 [+- 0.01], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 6.94, time.warmup: 2.34, time.bench: 4.60
UniformBitmapBenchmarkTest.timeGetIntersect: [measured 2 out of 3 rounds, threads: 1 (sequential)]
 round: 0.70 [+- 0.11], round.block: 0.00 [+- 0.00], round.gc: 0.00 [+- 0.00], GC.calls: 0, GC.time: 0.00, time.total: 2.01, time.warmup: 0.61, time.bench: 1.40
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 8.936 sec - in com.metamx.collections.bitmap.UniformBitmapBenchmarkTest